.github/workflows/r.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       run: |
         pip install -e $(git rev-parse --show-toplevel)
         H2O_VERSION=$(Rscript -e "print(packageVersion('h2o'))" | grep -Eo '[0-9][0-9.]+')
-        pip install xgboost tensorflow "h2o==$H2O_VERSION"
+        pip install xgboost tensorflow<2.11 "h2o==$H2O_VERSION"
```

Comment:
Can we add a TODO comment to remove the pinned TF installation and file a follow-up JIRA to track reverting this PR?

Reply:
Ticket already filed for next sprint (27035)

Reply:
and TODO added above run statement

Reply:
Awesome! Let's ship it :D